lms/templates/dashboard.html

```diff
@@ -103,12 +103,13 @@ <h2 class="header-courses">${_("My Courses")}</h2>
       </ul>
     % else:
       <section class="empty-dashboard-message">
-        <p>${_("Looks like you haven't enrolled in any courses yet.")}</p>
+        <p>${_("It looks like you aren't yet enrolled in any courses!")}</p>
```

Comment:
Let's review this in person tomorrow. My initial concern is around making this work for both openedx and edX contexts. 
I also would expect this messaging to carry less of a burden to convince and more to guide / redirect someone to our discovery process.
Does that make sense? 

Sent from Outlook Mobile




On Mon, Apr 11, 2016 at 2:06 PM -0700, "Sylvia Pearce" <notifications@github.com> wrote:












In lms/templates/dashboard.html:
> @@ -103,12 +103,13 @@ <h2 class="header-courses">${_("My Courses")}</h2>
>        </ul>
>      % else:
>        <section class="empty-dashboard-message">
> -        <p>${_("Looks like you haven't enrolled in any courses yet.")}</p>
> +        <p>${_("It looks like you aren't yet enrolled in any courses!")}</p>



Something like this?



EdX offers hundreds of courses on subjects from computer science and personal finance to advanced Italian and data analysis. Learn from the world’s best institutions today!



—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub